components/cli/vendor/github.com/docker/docker/pkg/term/termios_linux.go

```diff
@@ -18,20 +16,21 @@ type Termios unix.Termios
 // mode and returns the previous state of the terminal so that it can be
 // restored.
 func MakeRaw(fd uintptr) (*State, error) {
-	var oldState State
-	if _, _, err := unix.Syscall(unix.SYS_IOCTL, fd, getTermios, uintptr(unsafe.Pointer(&oldState.termios))); err != 0 {
+	termios, err := unix.IoctlGetTermios(int(fd), getTermios)
```

Comment:
Brought in by PR:
* https://github.com/moby/moby/pull/34135 Replace manually written wrappers by functions from x/sys/unix